apps/medon-fe/src/components/Navigation/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { useTranslation } from 'react-i18next';
+import {
+  NavContainer,
+  HeaderBlock,
+  Ul,
+  Dashboard,
+  Briefcase,
+  NavLinkStyled,
+  Profile,
+  Patient,
+  Help,
+  Logout,
+  UserBlock,
+  UserAvatar,
+  UserName,
+} from 'components/Navigation/styles';
+import profileImagePlaceholder from 'assets/images/Avatar.svg';
+import Logo from 'components/Logo';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from 'redux/hooks';
+import { getUserSelector } from 'redux/features/userSlice/userSelectors';
+import {
+  initialState,
+  setToken,
+  setUser,
+} from 'redux/features/userSlice/userSlice';
+import {navigation} from 'utils/constants/navigation'
```

Comment:
you dont configure prettier. compare with line 20

Reply:
fixed